specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs

```diff
@@ -44,18 +59,33 @@ private static IEnumerable<Type> Types
         .GetTypes()
         .Where(t => !t.IsAbstract && t.IsAssignableTo(typeof(DiagnosticAnalyzer)));
 
-    private static IEnumerable<DiagnosticDescriptor> Descriptors
+    private static IEnumerable<Rule> Descriptors
```

Comment:
I think `Rules` might now be a more descriptive name.

Reply:
Rules is not allowed, that's the name of the parent class, but `AllRules` is also better.